mailmap: V12.0.1 credits

swamireddy opened the pull request:
DNM: V12.0.1 credits - updated the .orgnizationmap and .mailmap files

swamireddy commented:
@dachary - Can you please review the above pull request?

dachary commented on .organizationmap:
```diff
@@ -483,9 +494,11 @@ The Linux Box <contact@linuxbox.com> Casey Bodley <casey@linuxbox.com>
 Ubuntu Kylin <contact@ubuntukylin.com> Min Chen <minchen@ubuntukylin.com>
 UMCloud <contact@umcloud.com> Jiaying Ren <mikulely@gmail.com>
 UMCloud <contact@umcloud.com> Rongze Zhu <zrzhit@gmail.com>
+UMCloud <contact@umcloud.com> ZVampirEM77 <enming.zhang@umcloud.com>
```
Enming Zhang ?

swamireddy replied:
Will update. Thanks

dachary commented on .organizationmap:
```diff
@@ -736,6 +756,7 @@ ZTE <contact@zte.com.cn> Luo Runbing <runsisi@zte.com.cn>
 ZTE <contact@zte.com.cn> Xie Xingguo <xie.xingguo@zte.com.cn>
 ZTE <contact@zte.com.cn> Shun Song <song.shun3@zte.com.cn>
 ZTE <contact@zte.com.cn> Song Baisen <song.baisen@zte.com.cn>
+ZTE <contact@zte.com.cn> tangwenjun3 <tang.wenjun3@zte.com.cn>
```
Tang Wenjun ?

swamireddy replied:
Will update. Thanks

swamireddy commented:
@dachary  - Updated as suggested. Thanks Swami

dachary commented:
When I run the script on your branch here is what I get:
<pre>
Number of lines added and removed, by authors
     1	 18275 Sage Weil <sweil@redhat.com>
     2	 15281 Jason Dillaman <dillaman@redhat.com>
     3	 10016 Chang Liu <liuchang0812@gmail.com>
     4	  9043 Yehuda Sadeh <ysadehwe@redhat.com>
     5	  8715 Abhishek Lekshmanan <abhishek@suse.com>
     6	  5488 Dongsheng Yang <dongsheng.yang@easystack.cn>
     7	  5350 John Spray <jspray@redhat.com>
     8	  2568 Mykola Golub <mgolub@mirantis.com>
     9	  2278 Kefu Chai <kchai@redhat.com>
    10	  2017 Venky Shankar <vshankar@redhat.com>
    11	  1823 Loic Dachary <ldachary@redhat.com>
    12	  1685 Victor Denisov <denisovenator@gmail.com>
    13	  1628 Nathan Cutler <ncutler@suse.com>
    14	  1468 Gui Hecheng <guihecheng@cmss.chinamobile.com>
    15	  1401 Igor Fedotov <ifedotov@mirantis.com>
    16	  1213 Haomai Wang <haomai@xsky.com>
    17	   875 Bartłomiej Święcki <bartlomiej.swiecki@corp.ovh.com>
    18	   680 Yan, Zheng <zyan@redhat.com>
    19	   665 Patrick Donnelly <pdonnell@redhat.com>
    20	   582 Yuri Weinstein <yuriw@redhat.com>
    21	   501 Amir Vadai <amir@vadai.me>
    22	   462 Samuel Just <sjust@redhat.com>
    23	   461 João Eduardo Luís <joao@suse.de>
    24	   445 Radoslaw Zarzynski <rzarzynski@mirantis.com>
    25	   417 Zhang Shaowen <zhangshaowen@cmss.chinamobile.com>
    26	   374 Matt Benjamin <mbenjamin@redhat.com>
    27	   356 Luo Runbing <runsisi@zte.com.cn>
    28	   272 Willem Jan Withagen <wjw@digiware.nl>
    29	   267 wei xiao <wei.xiao@linaro.org>
    30	   264 Ilya Dryomov <idryomov@redhat.com>
    31	   261 Casey Bodley <cbodley@redhat.com>
    32	   225 Xie Xingguo <xie.xingguo@zte.com.cn>
    33	   210 Pan Liu <pan.liu@istuary.com>
    34	   200 Yunchuan Wen <yunchuan.wen@kylin-cloud.com>
    35	   162 Adam Kupczyk <akupczyk@mirantis.com>
    36	   130 Song Baisen <song.baisen@zte.com.cn>
    37	   130 Barbora Ančincová <bara@redhat.com>
    38	   123 David Disseldorp <ddiss@suse.de>
    39	   123 Chuanhong Wang <wang.chuanhong@zte.com.cn>
    40	    98 Zack Cerza <zack@redhat.com>
    41	    97 Danny Al-Gaaf <danny.al-gaaf@bisect.de>
    42	    92 Jan Fajerski <jfajerski@suse.com>
    43	    90 Adam C. Emerson <aemerson@redhat.com>
    44	    82 Bassam Tabbara <bassam.tabbara@quantum.com>
    45	    73 wangzhengyong <wangzhengyong@cmss.chinamobile.com>
    46	    68 Ziye Yang <ziye.yang@intel.com>
    47	    68 David Zafman <dzafman@redhat.com>
    48	    66 Xiaoxi Chen <xiaoxchen@ebay.com>
    49	    65 Orit Wasserman <owasserm@redhat.com>
    50	    61 Vikhyat Umrao <vumrao@redhat.com>
    51	    57 Guillaume Abrioux <gabrioux@redhat.com>
    52	    56 Sarit Zubakov <saritz@mellanox.com>
    53	    54 Adir Lev <adirl@mellanox.com>
    54	    51 optimistyzy <optimistyzy@gmail.com>
    55	    49 Robin H. Johnson <robin.johnson@dreamhost.com>
    56	    49 Nikita Gerasimov <nikita.gerasimov@oracle.com>
    57	    48 Erwan Velu <erwan@redhat.com>
    58	    47 Wido den Hollander <wido@42on.com>
    59	    44 Ma Jianpeng <jianpeng.ma@intel.com>
    60	    36 Saumay Agrawal <saumay.agrawal@gmail.com>
    61	    32 Ken Dreyer <kdreyer@redhat.com>
    62	    30 Mingxin Liu <mingxin@xsky.com>
    63	    26 lu.shasha <lu.shasha@eisoo.com>
    64	    25 Zhiqiang Wang <zhiqiang@xsky.com>
    65	    23 Dan Mick <dmick@redhat.com>
    66	    23 Brad Hubbard <bhubbard@redhat.com>
    67	    21 Jiaying Ren <mikulely@gmail.com>
    68	    20 lvshuhua <lvshuhua@cmss.chinamobile.com>
    69	    19 Drunkard Zhang <gongfan193@gmail.com>
    70	    18 Greg Farnum <gfarnum@redhat.com>
    71	    15 Ali Maredia <amaredia@redhat.com>
    72	    12 Jing Wenjun <jingwenjun@cmss.chinamobile.com>
    73	    11 Zhang Zezhu <zhang.zezhu@zte.com.cn>
    74	    11 Dave Chen <wei.d.chen@intel.com>
    75	    11 Dan van der Ster <daniel.vanderster@cern.ch>
    76	    10 Zhi Zhang <willzzhang@tencent.com>
    77	    10 Ren Huanwen <ren.huanwen@zte.com.cn>
    78	    10 lihongjie <lihongjie@cmss.chinamobile.com>
    79	     9 tangwenjun <tang.wenjun3@zte.com.cn>
    80	     9 Pavan Rallabhandi <PRallabhandi@walmartlabs.com>
    81	     9 hrchu <hrchu@cht.com.tw>
    82	     8 Jenkins <jenkins@ceph.com>
    83	     7 Tamil Muthamizhan <tmuthami@redhat.com>
    84	     7 Michal Jarzabek <stiopa@gmail.com>
    85	     6 Tang Jin <tang.jin@istuary.com>
    86	     6 Tahia Khan <tahia.khan@gmail.com>
    87	     6 Sébastien Han <shan@redhat.com>
    88	     6 DanielBar-On <danielbo@mellanox.com>
    89	     6 Alexey Sheplyakov <asheplyakov@mirantis.com>
    90	     5 Ning Yao <yaoning@unitedstack.com>
    91	     5 Boris Ranto <branto@redhat.com>
    92	     4 Kyr Shatskyy <kyrylo.shatskyy@gmail.com>
    93	     4 Jie Wang <jie.wang@kylin-cloud.com>
    94	     4 haodong.tang <tanghaodong25@gmail.com>
    95	     4 Gu Zhongyan <guzhongyan@360.cn>
    96	     4 Ganesh Mahalingam <ganesh.mahalingam@intel.com>
    97	     4 Edwin F. Boza <eboza@espol.edu.ec>
    98	     3 Zhao Chao <zhaochao1984@gmail.com>
    99	     3 shiqi <1454927420@qq.com>
   100	     3 Li Wang <li.wang@kylin-cloud.com>
   101	     3 Josh Durgin <jdurgin@redhat.com>
   102	     2 ZVampirEM77 <enming.zhang@umcloud.com>
   103	     2 Yankun Li <liyankun@unitedstack.com>
   104	     2 Wei Jin <jinwei15@wanda.cn>
   105	     2 Taeksang Kim <voidbag@gmail.com>
   106	     2 Ruben Kerkhof <ruben@rubenkerkhof.com>
   107	     2 Ronak Jain <ronakjain@outlook.in>
   108	     2 Owen Synge <osynge@suse.com>
   109	     2 Michael Eischer <michael.eischer@fau.de>
   110	     2 Liu Yang <yippeetry@gmail.com>
   111	     2 hechuang <hechuang@xsky.com>
   112	     2 Gaurav Kumar Garg <garg.gaurav52@gmail.com>
   113	     2 craigchi <craig10624@gmail.com>
   114	     1 Shinobu Kinjo <shinobu@redhat.com>
   115	     1 Jos Collin <jcollin@redhat.com>
   116	     1 Henrik Korkuc <henrik@uber.com>

Number of lines added and removed, by organization
     1	 58112 Red Hat <contact@redhat.com>
     2	 11021 SUSE <contact@suse.com>
     3	 10026 Tencent <contact@tencent.com>
     4	  6267 Mirantis <contact@mirantis.com>
     5	  5488 Easystack <contact@easystack.cn>
     6	  2000 China Mobile <contact@chinamobile.com>
     7	  1270 XSky <contact@xsky.com>
     8	   875 OVH <ceph@ovh.net>
     9	   855 ZTE <contact@zte.com.cn>
    10	   663 Unaffiliated <no@organization.net>
    11	   272 Digiware <contact@digiware.nl>
    12	   267 Linaro <contact@linaro.org>
    13	   216 Istuary Innovation Group <contact@istuary.com>
    14	   207 Kylin Cloud <contact@kylin-cloud.com>
    15	   127 Intel <contact@intel.com>
    16	   116 Mellanox <contact@mellanox.com>
    17	    97 Deutsche Telekom <contact@telekom.de>
    18	    82 Quantum Corporation <info@quantum.com>
    19	    66 Ebay <contact@ebay.com>
    20	    49 Oracle <contact@oracle.com>
    21	    49 DreamHost <contact@dreamhost.com>
    22	    47 Wido 42on <contact@42on.com>
    23	    26 EISOO <contact@eisoo.com>
    24	    11 CERN <contact@cern.ch>
    25	     9 Walmart Labs <contact@walmartlabs.com>
    26	     9 tangwenjun <tang.wenjun3@zte.com.cn>
    27	     9 Chunghwa Telecom <contact@cht.com.tw>
    28	     7 UnitedStack <contact@unitedstack.com>
    29	     5 Eayun <contact@eayun.com>
    30	     4 ESPOL <contact@espol.edu.ec>
    31	     3 shiqi <1454927420@qq.com>
    32	     2 ZVampirEM77 <enming.zhang@umcloud.com>
    33	     2 Wanda ffan <contact@wanda.cn>
    34	     2 Inktank <contact@inktank.com>
    35	     2 FAU <contact@fau.de>
    36	     1 Uber <contact@uber.com>

Commits, by authors
     1	    309 Sage Weil <sweil@redhat.com>
     2	     74 Kefu Chai <kchai@redhat.com>
     3	     66 John Spray <jspray@redhat.com>
     4	     59 Loic Dachary <ldachary@redhat.com>
     5	     44 Yehuda Sadeh <ysadehwe@redhat.com>
     6	     44 Jason Dillaman <dillaman@redhat.com>
     7	     44 Dongsheng Yang <dongsheng.yang@easystack.cn>
     8	     31 Chang Liu <liuchang0812@gmail.com>
     9	     26 Yan, Zheng <zyan@redhat.com>
    10	     25 Patrick Donnelly <pdonnell@redhat.com>
    11	     24 Haomai Wang <haomai@xsky.com>
    12	     19 Gui Hecheng <guihecheng@cmss.chinamobile.com>
    13	     18 Matt Benjamin <mbenjamin@redhat.com>
    14	     17 Willem Jan Withagen <wjw@digiware.nl>
    15	     16 Nathan Cutler <ncutler@suse.com>
    16	     16 Danny Al-Gaaf <danny.al-gaaf@bisect.de>
    17	     14 Pan Liu <pan.liu@istuary.com>
    18	     14 Igor Fedotov <ifedotov@mirantis.com>
    19	     14 Casey Bodley <cbodley@redhat.com>
    20	     13 Xie Xingguo <xie.xingguo@zte.com.cn>
    21	     13 Mykola Golub <mgolub@mirantis.com>
    22	     11 Venky Shankar <vshankar@redhat.com>
    23	     10 Zack Cerza <zack@redhat.com>
    24	     10 Song Baisen <song.baisen@zte.com.cn>
    25	     10 Radoslaw Zarzynski <rzarzynski@mirantis.com>
    26	      9 Drunkard Zhang <gongfan193@gmail.com>
    27	      9 Adam C. Emerson <aemerson@redhat.com>
    28	      9 Abhishek Lekshmanan <abhishek@suse.com>
    29	      8 Zhang Shaowen <zhangshaowen@cmss.chinamobile.com>
    30	      8 wangzhengyong <wangzhengyong@cmss.chinamobile.com>
    31	      8 Samuel Just <sjust@redhat.com>
    32	      8 Ma Jianpeng <jianpeng.ma@intel.com>
    33	      8 Jiaying Ren <mikulely@gmail.com>
    34	      6 David Zafman <dzafman@redhat.com>
    35	      5 Sarit Zubakov <saritz@mellanox.com>
    36	      5 Mingxin Liu <mingxin@xsky.com>
    37	      5 João Eduardo Luís <joao@suse.de>
    38	      5 Dan Mick <dmick@redhat.com>
    39	      4 Yunchuan Wen <yunchuan.wen@kylin-cloud.com>
    40	      4 Xiaoxi Chen <xiaoxchen@ebay.com>
    41	      4 Vikhyat Umrao <vumrao@redhat.com>
    42	      4 Ilya Dryomov <idryomov@redhat.com>
    43	      4 Barbora Ančincová <bara@redhat.com>
    44	      4 Adam Kupczyk <akupczyk@mirantis.com>
    45	      3 Zhang Zezhu <zhang.zezhu@zte.com.cn>
    46	      3 Orit Wasserman <owasserm@redhat.com>
    47	      3 optimistyzy <optimistyzy@gmail.com>
    48	      3 Ken Dreyer <kdreyer@redhat.com>
    49	      3 Jing Wenjun <jingwenjun@cmss.chinamobile.com>
    50	      3 Greg Farnum <gfarnum@redhat.com>
    51	      3 David Disseldorp <ddiss@suse.de>
    52	      3 Brad Hubbard <bhubbard@redhat.com>
    53	      3 Boris Ranto <branto@redhat.com>
    54	      3 Adir Lev <adirl@mellanox.com>
    55	      2 Ziye Yang <ziye.yang@intel.com>
    56	      2 Zhiqiang Wang <zhiqiang@xsky.com>
    57	      2 Wido den Hollander <wido@42on.com>
    58	      2 tangwenjun <tang.wenjun3@zte.com.cn>
    59	      2 Tahia Khan <tahia.khan@gmail.com>
    60	      2 Saumay Agrawal <saumay.agrawal@gmail.com>
    61	      2 Robin H. Johnson <robin.johnson@dreamhost.com>
    62	      2 Pavan Rallabhandi <PRallabhandi@walmartlabs.com>
    63	      2 lvshuhua <lvshuhua@cmss.chinamobile.com>
    64	      2 lu.shasha <lu.shasha@eisoo.com>
    65	      2 Luo Runbing <runsisi@zte.com.cn>
    66	      2 Jie Wang <jie.wang@kylin-cloud.com>
    67	      2 Erwan Velu <erwan@redhat.com>
    68	      2 Edwin F. Boza <eboza@espol.edu.ec>
    69	      2 DanielBar-On <danielbo@mellanox.com>
    70	      2 Chuanhong Wang <wang.chuanhong@zte.com.cn>
    71	      2 Bartłomiej Święcki <bartlomiej.swiecki@corp.ovh.com>
    72	      2 Amir Vadai <amir@vadai.me>
    73	      1 ZVampirEM77 <enming.zhang@umcloud.com>
    74	      1 Zhi Zhang <willzzhang@tencent.com>
    75	      1 Zhao Chao <zhaochao1984@gmail.com>
    76	      1 Yuri Weinstein <yuriw@redhat.com>
    77	      1 Yankun Li <liyankun@unitedstack.com>
    78	      1 wei xiao <wei.xiao@linaro.org>
    79	      1 Wei Jin <jinwei15@wanda.cn>
    80	      1 Victor Denisov <denisovenator@gmail.com>
    81	      1 tangwenjun3 <tang.wenjun3@zte.com.cn>
    82	      1 Tang Jin <tang.jin@istuary.com>
    83	      1 Tamil Muthamizhan <tmuthami@redhat.com>
    84	      1 Taeksang Kim <voidbag@gmail.com>
    85	      1 shiqi <1454927420@qq.com>
    86	      1 Shinobu Kinjo <shinobu@redhat.com>
    87	      1 Sébastien Han <shan@redhat.com>
    88	      1 Ruben Kerkhof <ruben@rubenkerkhof.com>
    89	      1 Ronak Jain <ronakjain@outlook.in>
    90	      1 Ren Huanwen <ren.huanwen@zte.com.cn>
    91	      1 Owen Synge <osynge@suse.com>
    92	      1 Ning Yao <yaoning@unitedstack.com>
    93	      1 Nikita Gerasimov <nikita.gerasimov@oracle.com>
    94	      1 Michal Jarzabek <stiopa@gmail.com>
    95	      1 Michael Eischer <michael.eischer@fau.de>
    96	      1 Li Wang <li.wang@kylin-cloud.com>
    97	      1 Liu Yang <yippeetry@gmail.com>
    98	      1 lihongjie <lihongjie@cmss.chinamobile.com>
    99	      1 Kyr Shatskyy <kyrylo.shatskyy@gmail.com>
   100	      1 Josh Durgin <jdurgin@redhat.com>
   101	      1 Jos Collin <jcollin@redhat.com>
   102	      1 Jenkins <jenkins@ceph.com>
   103	      1 Jan Fajerski <jfajerski@suse.com>
   104	      1 hrchu <hrchu@cht.com.tw>
   105	      1 Henrik Korkuc <henrik@uber.com>
   106	      1 hechuang <hechuang@xsky.com>
   107	      1 haodong.tang <tanghaodong25@gmail.com>
   108	      1 Gu Zhongyan <guzhongyan@360.cn>
   109	      1 Guillaume Abrioux <gabrioux@redhat.com>
   110	      1 Gaurav Kumar Garg <garg.gaurav52@gmail.com>
   111	      1 Ganesh Mahalingam <ganesh.mahalingam@intel.com>
   112	      1 Dave Chen <wei.d.chen@intel.com>
   113	      1 Dan van der Ster <daniel.vanderster@cern.ch>
   114	      1 craigchi <craig10624@gmail.com>
   115	      1 Bassam Tabbara <bassam.tabbara@quantum.com>
   116	      1 Ali Maredia <amaredia@redhat.com>
   117	      1 Alexey Sheplyakov <asheplyakov@mirantis.com>

Commits, by organizations
     1	    765 Red Hat <contact@redhat.com>
     2	     44 Easystack <contact@easystack.cn>
     3	     43 Mirantis <contact@mirantis.com>
     4	     41 China Mobile <contact@chinamobile.com>
     5	     35 Unaffiliated <no@organization.net>
     6	     35 SUSE <contact@suse.com>
     7	     32 XSky <contact@xsky.com>
     8	     32 Tencent <contact@tencent.com>
     9	     31 ZTE <contact@zte.com.cn>
    10	     17 Digiware <contact@digiware.nl>
    11	     16 Deutsche Telekom <contact@telekom.de>
    12	     15 Istuary Innovation Group <contact@istuary.com>
    13	     12 Intel <contact@intel.com>
    14	     10 Mellanox <contact@mellanox.com>
    15	      7 Kylin Cloud <contact@kylin-cloud.com>
    16	      4 Ebay <contact@ebay.com>
    17	      2 Wido 42on <contact@42on.com>
    18	      2 Walmart Labs <contact@walmartlabs.com>
    19	      2 UnitedStack <contact@unitedstack.com>
    20	      2 tangwenjun <tang.wenjun3@zte.com.cn>
    21	      2 OVH <ceph@ovh.net>
    22	      2 ESPOL <contact@espol.edu.ec>
    23	      2 EISOO <contact@eisoo.com>
    24	      2 Eayun <contact@eayun.com>
    25	      2 DreamHost <contact@dreamhost.com>
    26	      1 ZVampirEM77 <enming.zhang@umcloud.com>
    27	      1 Wanda ffan <contact@wanda.cn>
    28	      1 Uber <contact@uber.com>
    29	      1 tangwenjun3 <tang.wenjun3@zte.com.cn>
    30	      1 shiqi <1454927420@qq.com>
    31	      1 Quantum Corporation <info@quantum.com>
    32	      1 Oracle <contact@oracle.com>
    33	      1 Linaro <contact@linaro.org>
    34	      1 Inktank <contact@inktank.com>
    35	      1 FAU <contact@fau.de>
    36	      1 Chunghwa Telecom <contact@cht.com.tw>
    37	      1 CERN <contact@cern.ch>

Reviews, by authors (one review spans multiple commits)
     1	    112 Sage Weil <sweil@redhat.com>
     2	     98 Kefu Chai <kchai@redhat.com>
     3	     39 Jason Dillaman <dillaman@redhat.com>
     4	     31 Casey Bodley <cbodley@redhat.com>
     5	     29 Josh Durgin <jdurgin@redhat.com>
     6	     24 Haomai Wang <haomai@xsky.com>
     7	     21 Mykola Golub <mgolub@mirantis.com>
     8	     18 Loic Dachary <ldachary@redhat.com>
     9	     17 John Spray <jspray@redhat.com>
    10	     16 Brad Hubbard <bhubbard@redhat.com>
    11	     12 Yehuda Sadeh <ysadehwe@redhat.com>
    12	     11 Yan, Zheng <zyan@redhat.com>
    13	     10 Greg Farnum <gfarnum@redhat.com>
    14	      9 Nathan Cutler <ncutler@suse.com>
    15	      9 Igor Fedotov <ifedotov@mirantis.com>
    16	      8 Adir Lev <adirl@mellanox.com>
    17	      8 Abhishek Lekshmanan <abhishek@suse.com>
    18	      6 Ken Dreyer <kdreyer@redhat.com>
    19	      5 Xie Xingguo <xie.xingguo@zte.com.cn>
    20	      4 Yuri Weinstein <yuriw@redhat.com>
    21	      4 Samuel Just <sjust@redhat.com>
    22	      4 Orit Wasserman <owasserm@redhat.com>
    23	      4 David Zafman <dzafman@redhat.com>
    24	      3 Matt Benjamin <mbenjamin@redhat.com>
    25	      3 Jos Collin <jcollin@redhat.com>
    26	      2 Yuri Weinstein <yweinstei@redhat.com>
    27	      2 Willem Jan Withagen <wjw@digiware.nl>
    28	      2 Radoslaw Zarzynski <rzarzynski@mirantis.com>
    29	      2 Piotr Dałek <piotr.dalek@ts.fujitsu.com>
    30	      2 Patrick Donnelly <pdonnell@redhat.com>
    31	      2 Pan Liu <pan.liu@istuary.com>
    32	      2 João Eduardo Luís <joao@suse.de>
    33	      2 Ilya Dryomov <idryomov@redhat.com>
    34	      2 Dan Mick <dmick@redhat.com>
    35	      2 Chang Liu <liuchang0812@gmail.com>
    36	      2 Boris Ranto <branto@redhat.com>
    37	      1 Zhiqiang Wang <zhiqiang@xsky.com>
    38	      1 Varada Kari <varada.kari@sandisk.com>
    39	      1 Sébastien Han <shan@redhat.com>
    40	      1 Nathan Cutler <nculter@suse.com>
    41	      1 John Wilkins <jwilkins@redhat.com>
    42	      1 Jiaying Ren <mikulely@gmail.com>
    43	      1 Ian Colle <icolle@redhat.com>
    44	      1 Dongsheng Yang <dongsheng.yang@easystack.cn>
    45	      1 Alexey Sheplyakov <asheplyakov@mirantis.com>
    46	      1 Adam C. Emerson <aemerson@redhat.com>

Reviews, by organizations (one review spans multiple commits)
     1	    430 Red Hat <contact@redhat.com>
     2	     33 Mirantis <contact@mirantis.com>
     3	     25 XSky <contact@xsky.com>
     4	     19 SUSE <contact@suse.com>
     5	      8 Mellanox <contact@mellanox.com>
     6	      5 ZTE <contact@zte.com.cn>
     7	      2 Yuri Weinstein <yweinstei@redhat.com>
     8	      2 Tencent <contact@tencent.com>
     9	      2 Istuary Innovation Group <contact@istuary.com>
    10	      2 Fujitsu <contact@fujitsu.com>
    11	      2 Digiware <contact@digiware.nl>
    12	      2 Cloudwatt <libre.licensing@cloudwatt.com>
    13	      1 Unaffiliated <no@organization.net>
    14	      1 SanDisk <contact@sandisk.com>
    15	      1 Nathan Cutler <nculter@suse.com>
    16	      1 Ian Colle <icolle@redhat.com>
    17	      1 Easystack <contact@easystack.cn>
</pre>

You can see people are mentionned in the Organization list (Nathan, Ian, tangwenjun etc.). Please check those carefully and update the .mailmap accordingly.

swamireddy commented:
Will update for below:

    26	     9 tangwenjun   // .mailmap change for name normalization
    31	     3 shiqi <1454927420@qq.com> // Need to add patch for this..
    32	     2 ZVampirEM77 // .mailmap change for name normalization


On Thu, Apr 20, 2017 at 6:10 PM, Loic Dachary <notifications@github.com>
wrote:

> When I run the script on your branch here is what I get:
>
> Number of lines added and removed, by authors
>      1	 18275 Sage Weil
>      2	 15281 Jason Dillaman
>      3	 10016 Chang Liu
>      4	  9043 Yehuda Sadeh
>      5	  8715 Abhishek Lekshmanan
>      6	  5488 Dongsheng Yang
>      7	  5350 John Spray
>      8	  2568 Mykola Golub
>      9	  2278 Kefu Chai
>     10	  2017 Venky Shankar
>     11	  1823 Loic Dachary
>     12	  1685 Victor Denisov
>     13	  1628 Nathan Cutler
>     14	  1468 Gui Hecheng
>     15	  1401 Igor Fedotov
>     16	  1213 Haomai Wang
>     17	   875 Bartłomiej Święcki
>     18	   680 Yan, Zheng
>     19	   665 Patrick Donnelly
>     20	   582 Yuri Weinstein
>     21	   501 Amir Vadai
>     22	   462 Samuel Just
>     23	   461 João Eduardo Luís
>     24	   445 Radoslaw Zarzynski
>     25	   417 Zhang Shaowen
>     26	   374 Matt Benjamin
>     27	   356 Luo Runbing
>     28	   272 Willem Jan Withagen
>     29	   267 wei xiao
>     30	   264 Ilya Dryomov
>     31	   261 Casey Bodley
>     32	   225 Xie Xingguo
>     33	   210 Pan Liu
>     34	   200 Yunchuan Wen
>     35	   162 Adam Kupczyk
>     36	   130 Song Baisen
>     37	   130 Barbora Ančincová
>     38	   123 David Disseldorp
>     39	   123 Chuanhong Wang
>     40	    98 Zack Cerza
>     41	    97 Danny Al-Gaaf
>     42	    92 Jan Fajerski
>     43	    90 Adam C. Emerson
>     44	    82 Bassam Tabbara
>     45	    73 wangzhengyong
>     46	    68 Ziye Yang
>     47	    68 David Zafman
>     48	    66 Xiaoxi Chen
>     49	    65 Orit Wasserman
>     50	    61 Vikhyat Umrao
>     51	    57 Guillaume Abrioux
>     52	    56 Sarit Zubakov
>     53	    54 Adir Lev
>     54	    51 optimistyzy
>     55	    49 Robin H. Johnson
>     56	    49 Nikita Gerasimov
>     57	    48 Erwan Velu
>     58	    47 Wido den Hollander
>     59	    44 Ma Jianpeng
>     60	    36 Saumay Agrawal
>     61	    32 Ken Dreyer
>     62	    30 Mingxin Liu
>     63	    26 lu.shasha
>     64	    25 Zhiqiang Wang
>     65	    23 Dan Mick
>     66	    23 Brad Hubbard
>     67	    21 Jiaying Ren
>     68	    20 lvshuhua
>     69	    19 Drunkard Zhang
>     70	    18 Greg Farnum
>     71	    15 Ali Maredia
>     72	    12 Jing Wenjun
>     73	    11 Zhang Zezhu
>     74	    11 Dave Chen
>     75	    11 Dan van der Ster
>     76	    10 Zhi Zhang
>     77	    10 Ren Huanwen
>     78	    10 lihongjie
>     79	     9 tangwenjun
>     80	     9 Pavan Rallabhandi
>     81	     9 hrchu
>     82	     8 Jenkins
>     83	     7 Tamil Muthamizhan
>     84	     7 Michal Jarzabek
>     85	     6 Tang Jin
>     86	     6 Tahia Khan
>     87	     6 Sébastien Han
>     88	     6 DanielBar-On
>     89	     6 Alexey Sheplyakov
>     90	     5 Ning Yao
>     91	     5 Boris Ranto
>     92	     4 Kyr Shatskyy
>     93	     4 Jie Wang
>     94	     4 haodong.tang
>     95	     4 Gu Zhongyan
>     96	     4 Ganesh Mahalingam
>     97	     4 Edwin F. Boza
>     98	     3 Zhao Chao
>     99	     3 shiqi <1454927420@qq.com>
>    100	     3 Li Wang
>    101	     3 Josh Durgin
>    102	     2 ZVampirEM77
>    103	     2 Yankun Li
>    104	     2 Wei Jin
>    105	     2 Taeksang Kim
>    106	     2 Ruben Kerkhof
>    107	     2 Ronak Jain
>    108	     2 Owen Synge
>    109	     2 Michael Eischer
>    110	     2 Liu Yang
>    111	     2 hechuang
>    112	     2 Gaurav Kumar Garg
>    113	     2 craigchi
>    114	     1 Shinobu Kinjo
>    115	     1 Jos Collin
>    116	     1 Henrik Korkuc
>
> Number of lines added and removed, by organization
>      1	 58112 Red Hat
>      2	 11021 SUSE
>      3	 10026 Tencent
>      4	  6267 Mirantis
>      5	  5488 Easystack
>      6	  2000 China Mobile
>      7	  1270 XSky
>      8	   875 OVH
>      9	   855 ZTE
>     10	   663 Unaffiliated
>     11	   272 Digiware
>     12	   267 Linaro
>     13	   216 Istuary Innovation Group
>     14	   207 Kylin Cloud
>     15	   127 Intel
>     16	   116 Mellanox
>     17	    97 Deutsche Telekom
>     18	    82 Quantum Corporation
>     19	    66 Ebay
>     20	    49 Oracle
>     21	    49 DreamHost
>     22	    47 Wido 42on
>     23	    26 EISOO
>     24	    11 CERN
>     25	     9 Walmart Labs
>     26	     9 tangwenjun
>     27	     9 Chunghwa Telecom
>     28	     7 UnitedStack
>     29	     5 Eayun
>     30	     4 ESPOL
>     31	     3 shiqi <1454927420@qq.com>
>     32	     2 ZVampirEM77
>     33	     2 Wanda ffan
>     34	     2 Inktank
>     35	     2 FAU
>     36	     1 Uber
>
> Commits, by authors
>      1	    309 Sage Weil
>      2	     74 Kefu Chai
>      3	     66 John Spray
>      4	     59 Loic Dachary
>      5	     44 Yehuda Sadeh
>      6	     44 Jason Dillaman
>      7	     44 Dongsheng Yang
>      8	     31 Chang Liu
>      9	     26 Yan, Zheng
>     10	     25 Patrick Donnelly
>     11	     24 Haomai Wang
>     12	     19 Gui Hecheng
>     13	     18 Matt Benjamin
>     14	     17 Willem Jan Withagen
>     15	     16 Nathan Cutler
>     16	     16 Danny Al-Gaaf
>     17	     14 Pan Liu
>     18	     14 Igor Fedotov
>     19	     14 Casey Bodley
>     20	     13 Xie Xingguo
>     21	     13 Mykola Golub
>     22	     11 Venky Shankar
>     23	     10 Zack Cerza
>     24	     10 Song Baisen
>     25	     10 Radoslaw Zarzynski
>     26	      9 Drunkard Zhang
>     27	      9 Adam C. Emerson
>     28	      9 Abhishek Lekshmanan
>     29	      8 Zhang Shaowen
>     30	      8 wangzhengyong
>     31	      8 Samuel Just
>     32	      8 Ma Jianpeng
>     33	      8 Jiaying Ren
>     34	      6 David Zafman
>     35	      5 Sarit Zubakov
>     36	      5 Mingxin Liu
>     37	      5 João Eduardo Luís
>     38	      5 Dan Mick
>     39	      4 Yunchuan Wen
>     40	      4 Xiaoxi Chen
>     41	      4 Vikhyat Umrao
>     42	      4 Ilya Dryomov
>     43	      4 Barbora Ančincová
>     44	      4 Adam Kupczyk
>     45	      3 Zhang Zezhu
>     46	      3 Orit Wasserman
>     47	      3 optimistyzy
>     48	      3 Ken Dreyer
>     49	      3 Jing Wenjun
>     50	      3 Greg Farnum
>     51	      3 David Disseldorp
>     52	      3 Brad Hubbard
>     53	      3 Boris Ranto
>     54	      3 Adir Lev
>     55	      2 Ziye Yang
>     56	      2 Zhiqiang Wang
>     57	      2 Wido den Hollander
>     58	      2 tangwenjun
>     59	      2 Tahia Khan
>     60	      2 Saumay Agrawal
>     61	      2 Robin H. Johnson
>     62	      2 Pavan Rallabhandi
>     63	      2 lvshuhua
>     64	      2 lu.shasha
>     65	      2 Luo Runbing
>     66	      2 Jie Wang
>     67	      2 Erwan Velu
>     68	      2 Edwin F. Boza
>     69	      2 DanielBar-On
>     70	      2 Chuanhong Wang
>     71	      2 Bartłomiej Święcki
>     72	      2 Amir Vadai
>     73	      1 ZVampirEM77
>     74	      1 Zhi Zhang
>     75	      1 Zhao Chao
>     76	      1 Yuri Weinstein
>     77	      1 Yankun Li
>     78	      1 wei xiao
>     79	      1 Wei Jin
>     80	      1 Victor Denisov
>     81	      1 tangwenjun3
>     82	      1 Tang Jin
>     83	      1 Tamil Muthamizhan
>     84	      1 Taeksang Kim
>     85	      1 shiqi <1454927420@qq.com>
>     86	      1 Shinobu Kinjo
>     87	      1 Sébastien Han
>     88	      1 Ruben Kerkhof
>     89	      1 Ronak Jain
>     90	      1 Ren Huanwen
>     91	      1 Owen Synge
>     92	      1 Ning Yao
>     93	      1 Nikita Gerasimov
>     94	      1 Michal Jarzabek
>     95	      1 Michael Eischer
>     96	      1 Li Wang
>     97	      1 Liu Yang
>     98	      1 lihongjie
>     99	      1 Kyr Shatskyy
>    100	      1 Josh Durgin
>    101	      1 Jos Collin
>    102	      1 Jenkins
>    103	      1 Jan Fajerski
>    104	      1 hrchu
>    105	      1 Henrik Korkuc
>    106	      1 hechuang
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph/pull/14479#issuecomment-295722161>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABb52Tl-FmcHkKEtKKImH0mtAeNEZh5xks5rx1JCgaJpZM4M7Z_T>
> .
>

swamireddy commented:
Fixed the tangwenjun and ZVvampirEM77's. And "shiqi <1454927420@qq.com>"
not fixed, because, it's causing the conflict, so I will fix this alone
separately after this PR merge.

On Thu, Apr 20, 2017 at 6:36 PM, M Ranga Swami Reddy <swamireddy@gmail.com>
wrote:

> Will update for below:
>
>     26	     9 tangwenjun   // .mailmap change for name normalization
>     31	     3 shiqi <1454927420@qq.com> // Need to add patch for this..
>     32	     2 ZVampirEM77 // .mailmap change for name normalization
>
>
> On Thu, Apr 20, 2017 at 6:10 PM, Loic Dachary <notifications@github.com>
> wrote:
>
>> When I run the script on your branch here is what I get:
>>
>> Number of lines added and removed, by authors
>>      1	 18275 Sage Weil
>>      2	 15281 Jason Dillaman
>>      3	 10016 Chang Liu
>>      4	  9043 Yehuda Sadeh
>>      5	  8715 Abhishek Lekshmanan
>>      6	  5488 Dongsheng Yang
>>      7	  5350 John Spray
>>      8	  2568 Mykola Golub
>>      9	  2278 Kefu Chai
>>     10	  2017 Venky Shankar
>>     11	  1823 Loic Dachary
>>     12	  1685 Victor Denisov
>>     13	  1628 Nathan Cutler
>>     14	  1468 Gui Hecheng
>>     15	  1401 Igor Fedotov
>>     16	  1213 Haomai Wang
>>     17	   875 Bartłomiej Święcki
>>     18	   680 Yan, Zheng
>>     19	   665 Patrick Donnelly
>>     20	   582 Yuri Weinstein
>>     21	   501 Amir Vadai
>>     22	   462 Samuel Just
>>     23	   461 João Eduardo Luís
>>     24	   445 Radoslaw Zarzynski
>>     25	   417 Zhang Shaowen
>>     26	   374 Matt Benjamin
>>     27	   356 Luo Runbing
>>     28	   272 Willem Jan Withagen
>>     29	   267 wei xiao
>>     30	   264 Ilya Dryomov
>>     31	   261 Casey Bodley
>>     32	   225 Xie Xingguo
>>     33	   210 Pan Liu
>>     34	   200 Yunchuan Wen
>>     35	   162 Adam Kupczyk
>>     36	   130 Song Baisen
>>     37	   130 Barbora Ančincová
>>     38	   123 David Disseldorp
>>     39	   123 Chuanhong Wang
>>     40	    98 Zack Cerza
>>     41	    97 Danny Al-Gaaf
>>     42	    92 Jan Fajerski
>>     43	    90 Adam C. Emerson
>>     44	    82 Bassam Tabbara
>>     45	    73 wangzhengyong
>>     46	    68 Ziye Yang
>>     47	    68 David Zafman
>>     48	    66 Xiaoxi Chen
>>     49	    65 Orit Wasserman
>>     50	    61 Vikhyat Umrao
>>     51	    57 Guillaume Abrioux
>>     52	    56 Sarit Zubakov
>>     53	    54 Adir Lev
>>     54	    51 optimistyzy
>>     55	    49 Robin H. Johnson
>>     56	    49 Nikita Gerasimov
>>     57	    48 Erwan Velu
>>     58	    47 Wido den Hollander
>>     59	    44 Ma Jianpeng
>>     60	    36 Saumay Agrawal
>>     61	    32 Ken Dreyer
>>     62	    30 Mingxin Liu
>>     63	    26 lu.shasha
>>     64	    25 Zhiqiang Wang
>>     65	    23 Dan Mick
>>     66	    23 Brad Hubbard
>>     67	    21 Jiaying Ren
>>     68	    20 lvshuhua
>>     69	    19 Drunkard Zhang
>>     70	    18 Greg Farnum
>>     71	    15 Ali Maredia
>>     72	    12 Jing Wenjun
>>     73	    11 Zhang Zezhu
>>     74	    11 Dave Chen
>>     75	    11 Dan van der Ster
>>     76	    10 Zhi Zhang
>>     77	    10 Ren Huanwen
>>     78	    10 lihongjie
>>     79	     9 tangwenjun
>>     80	     9 Pavan Rallabhandi
>>     81	     9 hrchu
>>     82	     8 Jenkins
>>     83	     7 Tamil Muthamizhan
>>     84	     7 Michal Jarzabek
>>     85	     6 Tang Jin
>>     86	     6 Tahia Khan
>>     87	     6 Sébastien Han
>>     88	     6 DanielBar-On
>>     89	     6 Alexey Sheplyakov
>>     90	     5 Ning Yao
>>     91	     5 Boris Ranto
>>     92	     4 Kyr Shatskyy
>>     93	     4 Jie Wang
>>     94	     4 haodong.tang
>>     95	     4 Gu Zhongyan
>>     96	     4 Ganesh Mahalingam
>>     97	     4 Edwin F. Boza
>>     98	     3 Zhao Chao
>>     99	     3 shiqi <1454927420@qq.com>
>>    100	     3 Li Wang
>>    101	     3 Josh Durgin
>>    102	     2 ZVampirEM77
>>    103	     2 Yankun Li
>>    104	     2 Wei Jin
>>    105	     2 Taeksang Kim
>>    106	     2 Ruben Kerkhof
>>    107	     2 Ronak Jain
>>    108	     2 Owen Synge
>>    109	     2 Michael Eischer
>>    110	     2 Liu Yang
>>    111	     2 hechuang
>>    112	     2 Gaurav Kumar Garg
>>    113	     2 craigchi
>>    114	     1 Shinobu Kinjo
>>    115	     1 Jos Collin
>>    116	     1 Henrik Korkuc
>>
>> Number of lines added and removed, by organization
>>      1	 58112 Red Hat
>>      2	 11021 SUSE
>>      3	 10026 Tencent
>>      4	  6267 Mirantis
>>      5	  5488 Easystack
>>      6	  2000 China Mobile
>>      7	  1270 XSky
>>      8	   875 OVH
>>      9	   855 ZTE
>>     10	   663 Unaffiliated
>>     11	   272 Digiware
>>     12	   267 Linaro
>>     13	   216 Istuary Innovation Group
>>     14	   207 Kylin Cloud
>>     15	   127 Intel
>>     16	   116 Mellanox
>>     17	    97 Deutsche Telekom
>>     18	    82 Quantum Corporation
>>     19	    66 Ebay
>>     20	    49 Oracle
>>     21	    49 DreamHost
>>     22	    47 Wido 42on
>>     23	    26 EISOO
>>     24	    11 CERN
>>     25	     9 Walmart Labs
>>     26	     9 tangwenjun
>>     27	     9 Chunghwa Telecom
>>     28	     7 UnitedStack
>>     29	     5 Eayun
>>     30	     4 ESPOL
>>     31	     3 shiqi <1454927420@qq.com>
>>     32	     2 ZVampirEM77
>>     33	     2 Wanda ffan
>>     34	     2 Inktank
>>     35	     2 FAU
>>     36	     1 Uber
>>
>> Commits, by authors
>>      1	    309 Sage Weil
>>      2	     74 Kefu Chai
>>      3	     66 John Spray
>>      4	     59 Loic Dachary
>>      5	     44 Yehuda Sadeh
>>      6	     44 Jason Dillaman
>>      7	     44 Dongsheng Yang
>>      8	     31 Chang Liu
>>      9	     26 Yan, Zheng
>>     10	     25 Patrick Donnelly
>>     11	     24 Haomai Wang
>>     12	     19 Gui Hecheng
>>     13	     18 Matt Benjamin
>>     14	     17 Willem Jan Withagen
>>     15	     16 Nathan Cutler
>>     16	     16 Danny Al-Gaaf
>>     17	     14 Pan Liu
>>     18	     14 Igor Fedotov
>>     19	     14 Casey Bodley
>>     20	     13 Xie Xingguo
>>     21	     13 Mykola Golub
>>     22	     11 Venky Shankar
>>     23	     10 Zack Cerza
>>     24	     10 Song Baisen
>>     25	     10 Radoslaw Zarzynski
>>     26	      9 Drunkard Zhang
>>     27	      9 Adam C. Emerson
>>     28	      9 Abhishek Lekshmanan
>>     29	      8 Zhang Shaowen
>>     30	      8 wangzhengyong
>>     31	      8 Samuel Just
>>     32	      8 Ma Jianpeng
>>     33	      8 Jiaying Ren
>>     34	      6 David Zafman
>>     35	      5 Sarit Zubakov
>>     36	      5 Mingxin Liu
>>     37	      5 João Eduardo Luís
>>     38	      5 Dan Mick
>>     39	      4 Yunchuan Wen
>>     40	      4 Xiaoxi Chen
>>     41	      4 Vikhyat Umrao
>>     42	      4 Ilya Dryomov
>>     43	      4 Barbora Ančincová
>>     44	      4 Adam Kupczyk
>>     45	      3 Zhang Zezhu
>>     46	      3 Orit Wasserman
>>     47	      3 optimistyzy
>>     48	      3 Ken Dreyer
>>     49	      3 Jing Wenjun
>>     50	      3 Greg Farnum
>>     51	      3 David Disseldorp
>>     52	      3 Brad Hubbard
>>     53	      3 Boris Ranto
>>     54	      3 Adir Lev
>>     55	      2 Ziye Yang
>>     56	      2 Zhiqiang Wang
>>     57	      2 Wido den Hollander
>>     58	      2 tangwenjun
>>     59	      2 Tahia Khan
>>     60	      2 Saumay Agrawal
>>     61	      2 Robin H. Johnson
>>     62	      2 Pavan Rallabhandi
>>     63	      2 lvshuhua
>>     64	      2 lu.shasha
>>     65	      2 Luo Runbing
>>     66	      2 Jie Wang
>>     67	      2 Erwan Velu
>>     68	      2 Edwin F. Boza
>>     69	      2 DanielBar-On
>>     70	      2 Chuanhong Wang
>>     71	      2 Bartłomiej Święcki
>>     72	      2 Amir Vadai
>>     73	      1 ZVampirEM77
>>     74	      1 Zhi Zhang
>>     75	      1 Zhao Chao
>>     76	      1 Yuri Weinstein
>>     77	      1 Yankun Li
>>     78	      1 wei xiao
>>     79	      1 Wei Jin
>>     80	      1 Victor Denisov
>>     81	      1 tangwenjun3
>>     82	      1 Tang Jin
>>     83	      1 Tamil Muthamizhan
>>     84	      1 Taeksang Kim
>>     85	      1 shiqi <1454927420@qq.com>
>>     86	      1 Shinobu Kinjo
>>     87	      1 Sébastien Han
>>     88	      1 Ruben Kerkhof
>>     89	      1 Ronak Jain
>>     90	      1 Ren Huanwen
>>     91	      1 Owen Synge
>>     92	      1 Ning Yao
>>     93	      1 Nikita Gerasimov
>>     94	      1 Michal Jarzabek
>>     95	      1 Michael Eischer
>>     96	      1 Li Wang
>>     97	      1 Liu Yang
>>     98	      1 lihongjie
>>     99	      1 Kyr Shatskyy
>>    100	      1 Josh Durgin
>>    101	      1 Jos Collin
>>    102	      1 Jenkins
>>    103	      1 Jan Fajerski
>>    104	      1 hrchu
>>    105	      1 Henrik Korkuc
>>    106	      1 hechuang
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ceph/ceph/pull/14479#issuecomment-295722161>, or mute
>> the thread
>> <https://github.com/notifications/unsubscribe-auth/ABb52Tl-FmcHkKEtKKImH0mtAeNEZh5xks5rx1JCgaJpZM4M7Z_T>
>> .
>>
>
>

swamireddy commented:
@dachary - Can you please check now? (expect "shiqi <1454927420@qq.com>" rest all fixed)

..
Commits by organizations
     1      765 Red Hat <contact@redhat.com>
     2       44 Easystack <contact@easystack.cn>
     3       43 Mirantis <contact@mirantis.com>
     4       41 China Mobile <contact@chinamobile.com>
     5       35 Unaffiliated <no@organization.net>
     6       35 SUSE <contact@suse.com>
     7       32 XSky <contact@xsky.com>
     8       32 Tencent <contact@tencent.com>
     9       31 ZTE <contact@zte.com.cn>
    10       17 Digiware <contact@digiware.nl>
    11       16 Deutsche Telekom <contact@telekom.de>
    12       15 Istuary Innovation Group <contact@istuary.com>
    13       12 Intel <contact@intel.com>
    14       10 Mellanox <contact@mellanox.com>
    15        7 Kylin Cloud <contact@kylin-cloud.com>
    16        4 Ebay <contact@ebay.com>
    17        3 Tang Wenjun <tang.wenjun3@zte.com.cn>
    18        2 Wido 42on <contact@42on.com>
    19        2 Walmart Labs <contact@walmartlabs.com>
    20        2 UnitedStack <contact@unitedstack.com>
    21        2 OVH <ceph@ovh.net>
    22        2 ESPOL <contact@espol.edu.ec>
    23        2 EISOO <contact@eisoo.com>
    24        2 Eayun <contact@eayun.com>
    25        2 DreamHost <contact@dreamhost.com>
    26        1 Wanda ffan <contact@wanda.cn>
    27        1 UMCloud <contact@umcloud.com>
    28        1 Uber <contact@uber.com>
    29        1 shiqi <1454927420@qq.com>
    30        1 Quantum Corporation <info@quantum.com>
    31        1 Oracle <contact@oracle.com>
    32        1 Linaro <contact@linaro.org>
    33        1 Inktank <contact@inktank.com>
    34        1 FAU <contact@fau.de>
    35        1 Chunghwa Telecom <contact@cht.com.tw>
    36        1 CERN <contact@cern.ch>
==

dachary commented:
These still need fixing:

<pre>
Number of lines added and removed, by organization
...
    26	     9 Tang Wenjun <tang.wenjun3@zte.com.cn>
    31	     3 shiqi <1454927420@qq.com>
Reviews, by organizations (one review spans multiple commits)
...
     7	      2 Yuri Weinstein <yweinstei@redhat.com>
    15	      1 Nathan Cutler <nculter@suse.com>
    16	      1 Ian Colle <icolle@redhat.com>
</pre>

You need to resolve the conflict before this is merged.

swamireddy commented:
@dachary - now looks clean.

dachary commented on .mailmap:
```diff
@@ -258,7 +259,7 @@ Min Chen <minchen@ubuntukylin.com>
 MingXin Liu <mingxin.liu@kylin-cloud.com> <mingxinliu@ubuntukylin.com>
 Mingyue Zhao <zhao.mingyue@h3c.com>
 Mykola Golub <mgolub@mirantis.com> <mgolub@zhuzha.mirantis.lviv.net>
-Nathan Cutler <ncutler@suse.com> 
```
You need to keep 
<pre>
Nathan Cutler &lt;ncutler@suse.com> 
</pre>
and add
<pre>
Nathan Cutler &lt;ncutler@suse.com> &lt;nculter@suse.com>
</pre>
after it.

swamireddy replied:
Updated as above.

dachary commented on .mailmap:
```diff
@@ -364,6 +365,7 @@ Sushma Gurram <sushma.gurram@sandisk.com>
 Swami Reddy <swami.reddy@ril.com> <swamireddy@gmail.com>
 Sylvain Munaut <s.munaut@whatever-company.com> <tnt@246tNt.com>
 Takeshi Miyamae <miyamae.takeshi@jp.fujitsu.com>
+Tang Wenjun <tang.wenjun3@zte.com.cn>
```
Duplicate Tang Wenjun

dachary commented on .mailmap:
```diff
@@ -372,6 +374,7 @@ Tamil Muthamizhan <tmuthami@redhat.com>
 Tamil Muthamizhan <tmuthami@redhat.com> <tmuthamizhan@MacBook-Air.local>
 Tamil Muthamizhan <tmuthami@redhat.com> <tamil@magna002.ceph.redhat.com>
 Tang Jin <tang.jin@istuary.com>
+Tang Wenjun <tang.wenjun3@zte.com.cn>
```
Duplicate Tang Wenjun

swamireddy replied:
Updated.

swamireddy replied:
@dachary - Hope, all fixed, can you please check?

dachary commented:
It looks good. Now please resolve the conflict.

swamireddy commented:
Updated the shiqi's conflicts.. Could you please check?

Thanks
Swami

On Sat, Apr 22, 2017 at 12:20 AM, Loic Dachary <notifications@github.com>
wrote:

> It looks good. Now please resolve the conflict.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph/pull/14479#issuecomment-296274087>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABb52TRlCG-xz26ljTr6c_ImwjbczEbFks5ryPpegaJpZM4M7Z_T>
> .
>

dachary commented:
The .organizationmap conflict is still here

swamireddy commented:
Conflicts gone from the CI test..."Henrik Korkuc" affiliation causing the
conflicts. I will submit the patch for "Henrik Korkuc" affliliation as a
next patch.

On Mon, Apr 24, 2017 at 11:57 AM, Loic Dachary <notifications@github.com>
wrote:

> The .organizationmap conflict is still here
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph/pull/14479#issuecomment-296537056>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABb52UoPTS1B8LT82KPNarFCWI9VbSWeks5rzEDqgaJpZM4M7Z_T>
> .
>

swamireddy commented:
check

dachary commented:
jenkins test this please
